test/013_oracles_composite.ts

```diff
@@ -101,4 +111,56 @@ describe('Oracles - Composite', function () {
     expect((await compositeMultiOracle.peek(bytes6ToBytes32(DAI), bytes6ToBytes32(USDC), WAD))[0]).to.equal(oneUSDC)
     expect((await compositeMultiOracle.peek(bytes6ToBytes32(USDC), bytes6ToBytes32(DAI), oneUSDC))[0]).to.equal(WAD)
   })
+
+  describe('Lido Oracle', () => {
```

Comment:
These test are redundant, although I understand you wanting to do them to make sure the combination works. I would move them to `016_oracles_lido.ts`, though.

Also, please rename `016_oracle_lido.ts` to `016_oracles_lido.ts` for consistency